DEVELOP.rst

```diff
@@ -45,16 +45,24 @@ Preparing a Release
 
 To create a new release, you must:
 
+- Backport your changes to the release branch x.y
```

Comment:
I think "release branch" is missleading. We've a stable branch and that should only receive bug-fixes. Features are in master. And if a new feature is released it's necessary to create a new stable branch.